src/figuresClasses.ts

```diff
@@ -1,19 +1,62 @@
+function round(num: number): number {
+  return Math.floor(num * 100) / 100;
+}
 export interface Figure {
```

Comment:
I suggest defining more types 
1. for possible color values (`red`, `green` or `blue`)
2. for shapes (`triangle`, `circle` or `rectangle`)